styles/_mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin desktop() {
+    @media (min-width: 600px) {
+        @content
+    }
+        
+    }
```

Comment:
I'd recommend cleaning this up a bit: adding an empty line at the end of the file, unindenting the last closing curly brace, and deleting line 5.